en/05_Contributing/17_Managing_Security_Issues.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Managing Security Issues
+summary: This document highlights how Silverstripe CMS security team handles security issue.
+iconBrand: shield-alt
+---
+
+# Managing Security Issues
+
+This document aims to provide a high level overview of how the Silverstripe CMS team handles security issues. Only members of the Silverstripe CMS security team can perform the actions outlined in this document.
```

Comment:
```suggestion
This document aims to provide a high level overview of how the Silverstripe CMS security team handles security issues. Only members of the Silverstripe CMS security team can perform the actions outlined in this document.
```
Lets consistently use the same phrase, otherwise it sounds like these are two different teams.